app/forms/update_user_mobile_form.rb

```diff
@@ -0,0 +1,41 @@
+class UpdateUserMobileForm
+  include ActiveModel::Model
+  include FormMobileValidator
+
+  attr_accessor :mobile
+  attr_reader :user
+
+  def persisted?
+    true
+  end
+
+  def initialize(user)
+    @user = user
+    self.mobile = @user.mobile
+  end
+
+  def submit(params)
+    process_attributes(params)
+
+    params.delete('mobile') if mobile_changed?
+
+    return unless valid?
+    @user.update(params)
```

Comment:
`process_attributes` is also doing the mobile formatting & has_changed check, so maybe we should leave?